### PR TITLE
[FLOC-2871] Quote vagrant floc 2871

### PR DESCRIPTION
**Pull request by adamtheturtle**
Fixes https://clusterhq.atlassian.net/browse/FLOC-2871

### CODE REVIEW

File: admin/test/test_release.py [hunk 1]
```diff
@@ -1874,6 +1874,33 @@ def test_metadata_content_type(self):
...
+
+        "/" and ":" are not escaped (these only appear in the protocol).
+        """
+        aws = FakeAWS(
+            routing_rules={},
+            s3_buckets={
+                self.target_bucket: {},
+            },
+        )
+
```
Comment (Azulinho) [inline admin/test/test_release.py]: not related to this PR, but I found interesting that there is a in-house mock implementation of AWS on ../aws.py.
Just out of curiosity, why wasn't https://github.com/spulec/moto used here ?

**Comment by Azulinho** [comment]
looks fine, tests are passing, berge it

**Comment by adamtheturtle** [comment]
Thanks @Azulinho - you're welcome to merge (my?) PRs if you're happy with them :smile:

File: admin/test/test_release.py (hunk 1 above)
Comment (adamtheturtle) [inline admin/test/test_release.py]: It has been suggested before.

**Pull request merged**